### PR TITLE
MOBILE-966 Timezone setting removed

### PR DESCRIPTION
**Pull request by mclomello**
Timezone setting removed because no make sense set this for a date without time definition

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DateField.java [hunk 1]
```diff
@@ -71,18 +70,14 @@ protected Date convertFromString(String stringValue) {
...
                 return new Date(Long.parseLong(stringValue));
             } else if (stringValue.length() - lastSeparator - 1 == 2) {
                 SimpleDateFormat simpleDateFormat = new SimpleDateFormat("MM/dd/yy", getCurrentLocale());
-                simpleDateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
                 return simpleDateFormat.parse(stringValue);
             } else {
                 SimpleDateFormat simpleDateFormat = new SimpleDateFormat("MM/dd/yyyy", getCurrentLocale());
-                simpleDateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
-
                 return simpleDateFormat.parse(stringValue);
```
Comment (phcp) [inline android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DateField.java:81]: SF. It needs an empty space before the line.

File: android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DateField.java [hunk 2]
```diff
@@ -71,18 +70,14 @@ protected Date convertFromString(String stringValue) {
 
             if (stringValue.contains("-")) {
                 SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd", getCurrentLocale());
-                simpleDateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
```
Comment (phcp) [inline android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DateField.java:74]: Maybe this class should be refactored, the piece of code above is repeated 4 times.
```
SimpleDateFormat simpleDateFormat = new SimpleDateFormat(<PATTERN>, getCurrentLocale());
return simpleDateFormat.parse(stringValue);
```

**Review by phcp — CHANGES_REQUESTED** [review]
Thanks for the PR! Please review the suggestions.

File: android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DateField.java (hunk 2 above)
Comment (victorlaerte) [reply android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DateField.java:74]: Extract in a method

**Review by victorlaerte — CHANGES_REQUESTED** [review]

**Comment by victorlaerte** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Review by victorlaerte — APPROVED** [review]

**Comment by victorlaerte** [comment]
Thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**